AUTHORS.md

```diff
@@ -0,0 +1,11 @@
+
+Author information
+------------------------------
+
+Copyright (c) 2021 - Monaco F. J. <monaco@usp.br>
+
+TexTron Snake source code received major contribution by
+the following authors:
+
+<!-- readme: contributors -start -->
+<!-- readme: contributors -end -->
```

Comment:
Theses lines make no sense in this PR, without the Workflow added by #35. I think it should be all in one PR.